✨ Add DonutsChart

Otavio-CB opened the pull request.

CauevSilv committed:
🏗️ feat: AddUser Method

CauevSilv committed:
🏗️ building: Permission add method.

pauloarantesmachado committed:
Merge pull request #43 from wiz-fatec/dev

:adhesive_bandage: Bug in the bronze metadata, the frequency didn't a…

CauevSilv committed:
🏗️ update: user_permission Relation

Otavio-CB committed:
💄 Update registration button alignment

CauevSilv committed:
✨ feat: Multiple Permissions to user

Modificado método para criar usuário permitindo a adição de vários níveis de permissões.
Também alterado o Front End para que as pemissões sejam 3 checkbox.

CauevSilv committed:
🏗️ merge with current dev

CauevSilv committed:
🏗️ fix : Permission fix after merge

Otavio-CB committed:
✨ Add DonutsChart

Otavio-CB committed:
🏗️ Update color chart

Cauanvmb committed:
:poop: branch created

Otavio-CB committed:
💄 Update margin-top and colors of the graph

gabmota8 committed:
Merge https://github.com/wiz-fatec/dom-rock-pipeline-configurator into feat/US0016-enterprise-dashboard

gabmota8 committed:
Merge branches 'feat/US0016-enterprise-dashboard' and 'feat/US0016-enterprise-dashboard' of https://github.com/wiz-fatec/dom-rock-pipeline-configurator into feat/US0016-enterprise-dashboard

CauevSilv committed:
🏗️ merge client with current dev

CauevSilv committed:
🏗️ Company DTO and get all comanies endpoint

Otavio-CB committed:
📈 Update color according to user

Otavio-CB committed:
Merge branch 'feat/US0016-enterprise-dashboard' of github.com:wiz-fatec/dom-rock-pipeline-configurator into feat/US0016-enterprise-dashboard

Otavio-CB committed:
👽️ Update route

Otavio-CB committed:
🐛 Merge sandbox dev

simulified commented:
sup
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 1000; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

ChavesVini committed:
:sparkles: added NumbersOfConfigurations method

ChavesVini committed:
:sparkles: added NumbersOfConfigurations method

CauevSilv committed:
🏗️ feat company integration

Cauanvmb committed:
:lipstick: barchart created

gabmota8 committed:
added the pie chart component and performed to its call on the Dashboard-View page

AlitaAmancio committed:
✨ Add DonutChart backend logic for "Usuario por Empresa"

ChavesVini committed:
:art: now the return of the method getConfigsByCompanies is in json

AlitaAmancio committed:
Merge with origin

AlitaAmancio committed:
Merge with origin

ChavesVini committed:
:sparkles:
integration of the "Configurações por Empresa" dash with the data received from the "getAllConfigsByCompany()" function

CauevSilv committed:
🏗️ merge with Spring Security branch

CauevSilv committed:
🏗️ fix UserDto and comapny reorder

CauevSilv committed:
🏗️ fix Migrations

CauevSilv committed:
🏗️ fix Company and Permission objects in User

CauevSilv committed:
🐛 fix Company and UserService relations

Cauanvmb committed:
"Updated SandboxView.vue with changes to template, script, and style sections"

CauevSilv committed:
🏗️ update Login Page

Cauanvmb committed:
Merge branch 'feat/US0016-enterprise-dashboard' into barChart

Cauanvmb committed:
Merge pull request #47 from wiz-fatec/barChart

Bar chart

pauloarantesmachado committed:
:construction: Company registration screen under construction

CauevSilv committed:
🐛 fix: JoinTables removed from MVC structures

CauevSilv committed:
🐛fix User entity annotations

CauevSilv committed:
🐛 fix User Login and SignUp methods

Cauanvmb committed:
:chart_with_upwards_trend: finished barchart dashboard

CauevSilv committed:
Merge branch 'feat/US0011-create-user' of https://github.com/wiz-fatec/dom-rock-pipeline-configurator into feat/US0011-create-user

Otavio-CB committed:
🚧 (feat): Implementing log table

pauloarantesmachado committed:
:construction: login integration and localStorage storage created

pauloarantesmachado committed:
:construction: add crossOrigin in the login controller

Otavio-CB committed:
💄 (feat): Add log screen

pauloarantesmachado committed:
:construction: add disabled button in screen home

AndreWakugawa committed:
🐛fix: getAllLogs access and usage

AndreWakugawa committed:
✨feat: http response code in logs

AndreWakugawa committed:
✨feat: login logs

AndreWakugawa committed:
♻️refactor: changed logic around new httpresponse on logs

also added exceptions on logging login attempts (without authorization)

and logging for other requests without authorization

AndreWakugawa committed:
♻️refactor: changed endpoints default to forbidden

anyone that wants to test can uncomment the permitAll line and comment the authenticated line

AndreWakugawa committed:
🚑️criticalfix: changed company reference from entity to cnpj

AndreWakugawa committed:
🐛fix: added missing responseCode on query -> json

Otavio-CB committed:
♻️ (refactor): Refactor data labels in log list

Otavio-CB committed:
Revert "♻️ (refactor): Refactor data labels in log list"

This reverts commit 285353ea2d40d9818a3aa9fe0fb4164318735388.

Otavio-CB committed:
♻️ (refactor): Refactor data labels in log list

Otavio-CB committed:
Merge branch 'dev' into feat/US0015-screen-log

CauevSilv committed:
✨ feat: Signup Screen

pauloarantesmachado committed:
:construction: Adding the token to the request

pauloarantesmachado committed:
:adhesive_bandage: Correction made for the request with the token

pauloarantesmachado committed:
:adhesive_bandage: Correction made for the request with the token

pauloarantesmachado committed:
:construction: Adding the token to the request

pauloarantesmachado committed:
:adhesive_bandage:Resolving conflict

Otavio-CB committed:
⚰️ (refactor): Remove unused code

Otavio-CB committed:
💄 (style): Update table css

pauloarantesmachado committed:
:adhesive_bandage: fix in disabled at button

Otavio-CB committed:
🔀 (merge): Merge branch create-user with screen-log

pauloarantesmachado committed:
:adhesive_bandage:adding CrossOrigin

Otavio-CB committed:
🚧 (refactor): Populating log table

Otavio-CB committed:
✅ (test): Update comments for test

Cauanvmb committed:
:construction: backend structured to use in the filter

AndreWakugawa committed:
♻️refactor: changed init admin creation so password is encrypted

now the initial admin can be accessed. previously the password wasn't encrypted so user was inaccessible

pauloarantesmachado committed:
:sparkles: finished on company screen

pauloarantesmachado committed:
:adhesive_bandage: fix the conflict

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

CauevSilv committed:
🐛 fix JWToken fix

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

CauevSilv committed:
:construction:  feat UserConfig rows

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

CauevSilv committed:
🐛fix UserCreation Json

CauevSilv committed:
🐛 fix createUser Json

pauloarantesmachado committed:
:adhesive_bandage: fix fixed the route of the domrock image and the reload of the company registration screen

Otavio-CB committed:
🔀 (merge): Merge branch create-user with screen-log

Otavio-CB committed:
🚚 (route): Add path to log table

Otavio-CB committed:
🛂 (feat): Update database password

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

pauloarantesmachado committed:
:adhesive_bandage: resolved some conflicts

Otavio-CB committed:
✨ (feat): Add functional log table

Otavio-CB committed:
♻️ (refactor): Refactor method extractCompanyCnpj

AndreWakugawa committed:
✨feat: "/list-view/{cnpj}" getConfigListByCnpj endpoint

AndreWakugawa committed:
Merge branch 'feat/US0011-create-user' of https://github.com/wiz-fatec/dom-rock-pipeline-configurator into feat/US0011-create-user

Cauanvmb committed:
:children_crossing: finished barChart

AndreWakugawa committed:
⚰️remove: dead code that came with merge on 5a67a9c

AndreWakugawa committed:
🩹minorfix: changed getAllLZByCnpj method name

pauloarantesmachado committed:
:adhesive_bandage: fix in the route of the domrock image and in the reload on the registration and company screen

pauloarantesmachado committed:
:adhesive_bandage: add scroll in screen createEnterprise

CauevSilv committed:
:construction: Populate Permissions SelectList on create user

CauevSilv committed:
✨ feat: full functional userEdit

pauloarantesmachado committed:
:adhesive_bandage: add cnpj in localStorage

AndreWakugawa committed:
♻️refactor: added cnpj field to lzconfig

pauloarantesmachado committed:
:necktie: added cnpj to the save settings endpoint

pauloarantesmachado committed:
:sparkles:added logout to the header component

pauloarantesmachado committed:
:adhesive_bandage: fix in header

pauloarantesmachado committed:
:coffin: remove the AdminAppHeader component

Cauanvmb committed:
Merge branch 'feat/US0011-create-user' into feat/US0016-enterprise-dashboard

pauloarantesmachado committed:
:adhesive_bandage: fix in pipeLine

AndreWakugawa committed:
♻️refactor: added cnpj to silver filter

pauloarantesmachado committed:
:adhesive_bandage: fix in company enterprise screen

AndreWakugawa committed:
🚑️fix: remove cnpj from admin & remove 1st company

pauloarantesmachado committed:
:adhesive_bandage: fix in conflict

CauevSilv committed:
🚑️ fix: Create User with the same email

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

# Conflicts:
#	web-client/src/components/AppHeader.vue
#	web-client/src/components/lz-config/LZUploadSection.vue
#	web-client/src/components/silver/ColumnSelectionSilver.vue
#	web-client/src/router/index.ts
#	web-client/src/views/SandboxView.vue
#	web-server/src/main/java/com/domrock/configurator/Controller/PermissionController.java
#	web-server/src/main/java/com/domrock/configurator/Services/CompanyService.java
#	web-server/src/main/java/com/domrock/configurator/Services/PermissionService.java
#	web-server/src/main/java/com/domrock/configurator/Services/UserService.java

AndreWakugawa committed:
⏪️revert: added cnpj back to admin

Otavio-CB committed:
Merge branch 'refs/heads/feat/US0011-create-user' into feat/US0015-screen-log

Otavio-CB committed:
🔀 (merge): Merge branch create-user with screen-log

Otavio-CB committed:
Merge pull request #48 from wiz-fatec/feat/US0015-screen-log

Feat/us0015 screen log

AndreWakugawa committed:
🩹smallfix: changed anyRequest from permitAll to authenticated

CauevSilv committed:
:necktie: update: removed SuperUser from edit

pauloarantesmachado committed:
:adhesive_bandage: fix conflict
